src/commands/appSettings/filterDownloadAppSettings.ts

```diff
@@ -11,42 +11,75 @@ import { localize } from "../../localize";
 export async function filterDownloadAppSettings(context: IActionContext, sourceSettings: { [key: string]: string }, destinationSettings: { [key: string]: string }, destinationName: string): Promise<void> {
     let suppressPrompt: boolean = false;
     let overwriteSetting: boolean = false;
+    let suppressPromptOfSettingsToIgnore = false;
+    let overwriteSettingsToIgnore: boolean = false;
 
     const addedKeys: string[] = [];
     const updatedKeys: string[] = [];
     const userIgnoredKeys: string[] = [];
     const matchingKeys: string[] = [];
+    const securitySettingsIgnored: string[] = [];
+    const listOfSettingsToIgnore: string[] = ["AzureWebJobsStorage", "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING", "WEBSITE_CONTENTSHARE"];
 
     for (const key of Object.keys(sourceSettings)) {
-        if (destinationSettings[key] === undefined) {
-            addedKeys.push(key);
-            destinationSettings[key] = sourceSettings[key];
-        } else if (destinationSettings[key] === sourceSettings[key]) {
-            matchingKeys.push(key);
-        } else if (sourceSettings[key]) { // ignore empty settings
-            if (!suppressPrompt) {
+        if (listOfSettingsToIgnore.includes(key)) {
+            if (!suppressPromptOfSettingsToIgnore) {
                 const yesToAll: vscode.MessageItem = { title: localize('yesToAll', 'Yes to all') };
```

Comment:
`yesToAll` and 'noToAll` seem to be declared twice.. anyway we can refactor to have it declared once a the top of the function?

Reply:
Sure, I can do that.